### ISSUE TITLE
Stuck on caching latents

### ISSUE THREAD

**Issue by DReykaaa** [serpotapov/Kohya_ss-GUI-LoRA-Portable#12]
HI!

Iv got stuck on caching latents with 0% of using GPU RAM. Im runing on RTX 3060 Laptop GPU
Heres the full log. 

Folder 100_Gaby: 18 images found
Folder 100_Gaby: 1800 steps
max_train_steps = 900
stop_text_encoder_training = 0
lr_warmup_steps = 0
accelerate launch --num_cpu_threads_per_process=2 "train_network.py" --pretrained_model_name_or_path="C:/Lora/Models/deliberate_v2.safetensors" --train_data_dir="C:/Lora/Models/ready" --resolution=512,512 --output_dir="C:/Lora/Models/ready" --logging_dir="" --network_alpha="128" --save_model_as=ckpt --network_module=networks.lora --text_encoder_lr=5e-5 --unet_lr=0.0001 --network_dim=128 --output_name="Gaby" --lr_scheduler_num_cycles="1" --learning_rate="0.0001" --lr_scheduler="constant" --train_batch_size="2" --max_train_steps="900" --save_every_n_epochs="1" --mixed_precision="bf16" --save_precision="bf16" --seed="1" --cache_latents --optimizer_type="AdamW8bit" --max_data_loader_n_workers="1" --clip_skip=2 --bucket_reso_steps=64 --xformers --bucket_no_upscale
2023-04-12 00:53:04.485524: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'cudart64_110.dll'; dlerror: cudart64_110.dll not found
2023-04-12 00:53:04.485661: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2023-04-12 00:53:19.147090: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'cudart64_110.dll'; dlerror: cudart64_110.dll not found
2023-04-12 00:53:19.147180: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
Could not find module 'C:\Lora\venv\Lib\site-packages\xformers\_C.pyd' (or one of its dependencies). Try using the full path with constructor syntax.
WARNING:root:WARNING: Could not find module 'C:\Lora\venv\Lib\site-packages\xformers\_C.pyd' (or one of its dependencies). Try using the full path with constructor syntax.
Need to compile C++ extensions to get sparse attention suport. Please run python setup.py build develop
prepare tokenizer
Use DreamBooth method.
prepare images.
found directory C:\Lora\Models\ready\100_Gaby contains 18 image files
1800 train images with repeating.
0 reg images.
no regularization images / 正則化画像が見つかりませんでした
[Dataset 0]
  batch_size: 2
  resolution: (512, 512)
  enable_bucket: False

  [Subset 0 of Dataset 0]
    image_dir: "C:\Lora\Models\ready\100_Gaby"
    image_count: 18
    num_repeats: 100
    shuffle_caption: False
    keep_tokens: 0
    caption_dropout_rate: 0.0
    caption_dropout_every_n_epoches: 0
    caption_tag_dropout_rate: 0.0
    color_aug: False
    flip_aug: False
    face_crop_aug_range: None
    random_crop: False
    token_warmup_min: 1,
    token_warmup_step: 0,
    is_reg: False
    class_tokens: Gaby
    caption_extension: .caption

[Dataset 0]
loading image sizes.
100%|█████████████████████████████████████████████████████████████████████████████████| 18/18 [00:00<00:00, 248.12it/s]
prepare dataset
prepare accelerator
Using accelerator 0.15.0 or above.
loading model for process 0/1
load StableDiffusion checkpoint
C:\Lora\venv\lib\site-packages\safetensors\torch.py:98: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()
  with safe_open(filename, framework="pt", device=device) as f:
C:\Lora\venv\lib\site-packages\torch\_utils.py:776: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()
  return self.fget.__get__(instance, owner)()
C:\Lora\venv\lib\site-packages\torch\storage.py:899: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()
  storage = cls(wrap_storage=untyped_storage)
loading u-net: <All keys matched successfully>
loading vae: <All keys matched successfully>
loading text encoder: <All keys matched successfully>
Replace CrossAttention.forward to use xformers
[Dataset 0]
caching latents.
  0%|

**Comment by serpotapov** [comment]
Hey 🖐
You need to reinstall portable version.
Some files didn't download correctly.
